Client/Frontend/Strings.swift

```diff
@@ -640,7 +638,7 @@ extension Strings {
     public static let TrackerProtectionAlertButton =  MZLocalizedString("Settings.TrackingProtection.Alert.Button", value: "OK, Got It", comment: "Dismiss button for the tracker protection alert.")
 }
 
-// Tracking Protection menu
+// MARK: - Tracking Protection men
```

Comment:
You probably meant "menu" 😅

Reply:
LOL. Yep. Why did these get cut off! Weird!

Reply:
```suggestion
// MARK: - Tracking Protection menu
```